src/modules/blog/entities/blog-category.entity.ts

```diff
@@ -0,0 +1,12 @@
+import { Entity, Column, OneToMany } from 'typeorm';
+import { AbstractBaseEntity } from '../../../entities/base.entity';
+import { Blog } from './blog.entity';
+
+@Entity()
+export class BlogCategory extends AbstractBaseEntity {
```

Comment:
Rename this to BlogPostCategory and update others accordingly

Reply:
fix